Src/BootCamp.Chapter/Checks.cs

```diff
@@ -1,39 +1,216 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     // This class is used to have a freedom of design, but with tests applied.
+
     public static class Checks
     {
         public static string FindMostPoorPerson(string[] peopleAndBalances)
         {
-            return "";
+            if (Testers.IsThisStringArrayValid(peopleAndBalances))
+            {
+                return "N/A.";
+            }
+            StringBuilder poorestName = new StringBuilder();
+            decimal amount = 0;
+            int personCount = 1;
+
+            //TODO The body of the for loop could be refactored into a function (or a few) so that it is easier to skim through the bigger function.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)